<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetSecurity</class>
 <widget class="QWidget" name="SettingsWidgetSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>303</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="clearClipboardCheckBox">
     <property name="text">
      <string>Clear clipboard after</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="clearClipboardSpinBox">
     <property name="suffix">
      <string> sec</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="lockDatabaseIdleCheckBox">
     <property name="text">
      <string>Lock databases after inactivity of</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="lockDatabaseIdleSpinBox">
     <property name="suffix">
      <string> sec</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clearClipboardCheckBox</tabstop>
  <tabstop>clearClipboardSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
